src/cnxml/serialize.js

```diff
@@ -142,6 +143,30 @@ function list(obj, children) {
 }
 
 
+/**
+ * Serializer for source elements.
+ */
+function source(obj, children) {
+    const parser = new DOMParser();
+    const xmlDoc = parser.parseFromString(obj.getText().trim(), 'text/xml')
+    let el = document.createElement('template')
+    el.innerHTML = xmlDoc.documentElement.outerHTML
```

Comment:
What you are doing here is, in order:

1. Deserialize XML (with `DOMParser`)

2. Serialize it as HTML (with `.outerHTML`)

3. Deserialize it as HTML (with `.innerHTML =`)

4. Use this HTML fragment as XML

why?

Reply:
How you would do this? I've tried few other options and that was the one that I found working.

Reply:
```
return new DOMParser().parseFromSting(obj.getText().trim(), 'application/xml`).documentElement
```